include/sys/vdev_raidz.h

```diff
@@ -84,6 +86,11 @@ typedef struct vdev_raidz_expand {
 
 	uint64_t vre_offset_pertxg[TXG_SIZE];
 
+	/*
+	 * Last reflow txg per attached device.
+	 */
+	avl_tree_t vre_txgs;
```

Comment:
The vdev_raidz_expand_t is intended to track the state of an in-progress expansion.  Since this AVL applies to any expanded raidz vdevs, I think it belongs in the vdev_raidz_t. (similar to how the logical_width is in the vdev_raidz_t)